src/microsoft-trydotnet-editor/src/index.css

```diff
@@ -1,5 +1,5 @@
 body {
-	width: 800px;
-	height: 600px;
+	width: auto;
+	height: auto;
 	border: 1px solid #ccc;
```

Comment:
Suggestion:
If you update this to add `margin: 0;` and then add padding to `.monaco-editor` then it will get rid of that white border in dark mode.

```
.body {
	width: auto;
	height: auto;
	border: 1px solid #ccc;
    margin: 0;
 }
 
.monaco-editor {
   padding: 0.5 rem; // 8px
}
```

**Before**
![image](https://github.com/dotnet/try/assets/38262647/8e5542bc-6051-4e06-bff6-46763ad0674a)


**After**
![image](https://github.com/dotnet/try/assets/38262647/28053b5c-91ca-4097-9504-be0d3f255e69)

Reply:
thank you!!